pennylane/templates/broadcast.py

```diff
@@ -23,9 +23,11 @@
 from collections import Iterable
 
 from pennylane.templates.decorator import template
-from pennylane.templates.utils import check_wires, check_type, get_shape, check_is_in_options
+from pennylane.templates.utils import check_type, get_shape, check_is_in_options
+from pennylane.wires import Wires
```

Comment:
After a very brief intro, really like the `Wires` class so far! :fire: :heart_eyes:

Reply:
Awesome